java/com/facebook/yoga/YogaNode.java

```diff
@@ -310,6 +310,18 @@ public void setOverflow(YogaOverflow overflow) {
     jni_YGNodeStyleSetOverflow(mNativePointer, overflow.intValue());
   }
 
+  private native int jni_YGNodeStyleGetDisplay(long nativePointer);
+  @Override
+  public YogaDisplay getDisplay() {
+    return YogaDisplay.values()[jni_YGNodeStyleGetDisplay(mNativePointer)];
```

Comment:
please use the generated fromInt() method on `YogaDisplay`. We should be doing the same in other places (i'll push a commit for that now, don't worry about it)